app/views/editions/non_manage_live_history_mode.html.erb

```diff
@@ -0,0 +1,8 @@
+<% content_for :back_link, render_back_link(href: document_path(@edition.document)) %>
+<% content_for :title, t("editions.non_manage_live_history_mode_permission.title", title: @edition.title) %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= render_govspeak(File.read("app/views/editions/non_manage_live_history_mode.govspeak.md")) %>
+  </div>
+</div>
```

Comment:
I wonder if we can make use of the errors partial for this to save duplication.

e.g.
```
<% content_for :back_link, render_back_link(href: document_path(@edition.document)) %>
<%= render partial: "errors/error",
  locals: {
    title: t("editions.non_manage_live_history_mode_permission.title", title: @edition.title),
    body_govspeak: File.read("app/views/editions/non_manage_live_history_mode.govspeak.md"),
  } %>
```

Reply:
I've decided to keep as is due to all controllers using the same template meaning less clutter.